doozerlib/rpm_builder.py

```diff
@@ -355,6 +356,8 @@ async def _populate_specfile_async(
                 lines[i] = f"%setup -q -n {rpm.config.name}-{rpm.version}\n"
             elif line.startswith("%autosetup"):
                 lines[i] = f"%autosetup -S git -n {rpm.config.name}-{rpm.version} -p1\n"
+            elif line.startswith("%changelog"):
+                lines[i] = f"{lines[i].strip()}\nUpdate to source commit {commit_url}\n"
```

Comment:
The format of changelog message doesn't seem correct. Please follow https://rpm-packaging-guide.github.io/